apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -107,15 +107,14 @@ public sealed class CloudTrace : IDisposable
             new TraceHeaderPropagatingHandler(() => Tracer);
 
         private CloudTrace(string projectId, TraceConfiguration config = null, TraceServiceClient client = null,
-            Func<HttpRequest, bool?> traceFallbackPredicate = null)
+            TraceDecisionPredicate traceFallbackPredicate = null)
         {
             GaxPreconditions.CheckNotNull(projectId, nameof(projectId));
 
             // Create the default values if not set.
             client = client ?? TraceServiceClient.Create();
-            config = config ?? TraceConfiguration.Create();
-
-            _traceFallbackPredicate = traceFallbackPredicate;
+            config = config ?? TraceConfiguration.Create(); 
+            _traceFallbackPredicate = traceFallbackPredicate ?? TraceDecisionPredicate.Create();
```

Comment:
Given that TraceDecisionPredicate is immutable, make this `TraceDecisionPredicate.Default` as a static property?